.coveragerc

```diff
@@ -696,6 +696,7 @@ omit =
     homeassistant/components/lcn/climate.py
     homeassistant/components/lcn/helpers.py
     homeassistant/components/lcn/services.py
+    homeassistant/components/lcn/websocket.py
```

Comment:
The integration already has tests, I think tests should be added for the websocket handlers.

Reply:
I added the tests.